tools/dump.cc

```diff
@@ -69,12 +80,24 @@ void RunMain(int argc, char** argv) {
             // ONNX test directory.
             DumpONNX(filename + "/model.onnx", args);
 
+            std::vector<std::string> filenames;
+#if _WIN32
+            fs::directory_iterator iter(filename);
+
+            std::regex filter(filename + "/*/*.pb");
```

Comment:
I don't think this regexp is equivalent to the original code. Maybe something like `"/[^/]+/[^/]+\\.pb"` ?